controller.go

```diff
@@ -476,7 +476,7 @@ func (c *controller) ID() string {
 
 func (c *controller) BuiltinDrivers() []string {
 	drivers := []string{}
-	for _, i := range getInitializers() {
+	for _, i := range getInitializers(c.cfg.Daemon.Experimental) {
```

Comment:
Can we instead do something like:
```
initializers := getInitializers()
if c.cfg.Daemon.Experimental {
    initializers = append(initializers, additionalDrivers()...)
}
for _, i := range initializers {
...
```
and remove the call to `additionalDrivers()` from inside `getInitializers()`